repo/fsrepo/fsrepo.go

```diff
@@ -164,30 +180,17 @@ func Init(repoPath string, conf *config.Config) error {
 
 // Remove recursively removes the FSRepo at |path|.
 func Remove(repoPath string) error {
-	repoPath = path.Clean(repoPath)
```

Comment:
@tv42 ahh didnt see that there. But, wait, that means that:

`fsrepo.Open(p)` and `fsrepo.Remove(p)` may do different things depending on how the user's call them or `p`'s value. i think they should probably be symmetric, both sanitize input.